src/server/table.h

```diff
@@ -92,17 +89,24 @@ class LockTable {
   bool Acquire(std::string_view key, IntentLock::Mode mode);
   void Release(std::string_view key, IntentLock::Mode mode);
```

Comment:
Here's a proposal: have overloads for both of these functions, the existing one (accepting a `string_view`), and another one accepting a fingerprint.
Then have the `string_view` version do the hash and call the overloaded one. This will save you the trouble of adding them in the future PR you plan to add.

Reply:
I already wrote this in another branch. I prefer adding them in the next PR but I did like you suggested.